.eslintrc.json

```diff
@@ -9,11 +9,16 @@
     "ecmaVersion": 2020,
     "sourceType": "module"
   },
-  "plugins": ["eslint-plugin-no-only-tests", "@typescript-eslint"],
+  "plugins": [
+    "eslint-plugin-import",
+    "eslint-plugin-no-only-tests",
+    "@typescript-eslint"
+  ],
   "env": {
     "browser": true
   },
   "rules": {
+    "import/extensions": ["error", "always", {"mjs": "never"}],
```

Comment:
The `.mjs` exclusion is for imports of `uvu/assert`. The TypeScript version we're using doesn't resolve `uvu/assert.mjs`. That might be fixed in a newer version.

Reply:
Weird... locally this fixes the problems with `uvu/assert`. On CI it doesn't. 🤷‍♂️

Reply:
I can repro the issue locally by doing:

```
git clone git@github.com:lit/lit.git
git checkout extensions
npm ci
npm run lint
```

Reply:
Well, I couldn't get whatever was causing things to pass locally to work on CI, so I just eslint-ignore'ed those imports.